src/main/java/org/rcsb/cif/model/FloatColumn.java

```diff
@@ -70,4 +68,51 @@ protected String getBinaryStringData(int row) {
     public double[] getBinaryData() {
         return binaryData;
     }
+
+    /**
+     * Uses Double.MIN_VALUE for not present and Double.MAX_VALUE for unknown
```

Comment:
Representing missing/unknown values is worth a consideration. However, that behavior can be easily implemented by using `getValueKind`. I don't think that there is an all-purpose solution by representing them by actual numbers. `Integer.MIN_VALUE` and `Integer.MAX_VALUE` basically become magic numbers in text/binary representation and are still difficult to handle.

Do you see a use case for a `getUnmaskedData` method, @BobHanson?